<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com)
     Commit Version: 1.5.0-0-g9aa0fdb  Build Version: 1.5.7004.21443
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot name="quadruped_model">



  <link
    name="base">
    <inertial>
      <origin
        xyz="0.006699 -0.000736 0.001288"
        rpy="0 0 0" />
      <mass
        value="27.801" />
      <inertia
        ixx="0.263893"
        ixy="0"
        ixz="0"
        iyy="2.018032"
        iyz="0"
        izz="2.056458" />
    </inertial>
  </link>
  <link
    name="LF_HIP">
    <inertial>
      <origin
        xyz="0.000688 0.10488 -0.003776"
        rpy="0 0 0" />
      <mass
        value="2.9231" />
      <inertia
        ixx="0.004820"
        ixy="0"
        ixz="0"
        iyy="0.007868"
        iyz="0"
        izz="0.008241" />
    </inertial>
  </link>
  <joint
    name="LF_HAA"
    type="revolute">
    <origin
      xyz="0.427 0.075 -0.0095"
      rpy="0 1.5708 0" />
    <parent
      link="base" />
    <child
      link="LF_HIP" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="LF_THIGH">
    <inertial>
      <origin
        xyz="0.18475 3.2E-05 0.12906"
        rpy="0 0 0" />
      <mass
        value="4.2164" />
      <inertia
        ixx="0.006286"
        ixy="0"
        ixz="0"
        iyy="0.028156"
        iyz="0"
        izz="0.029242" />
    </inertial>
  </link>
  <joint
    name="LF_HFE"
    type="revolute">
    <origin
      xyz="0 0 0"
      rpy="-1.5708 0 0" />
    <parent
      link="LF_HIP" />
    <child
      link="LF_THIGH" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="LF_SHANK">
    <inertial>
      <origin
        xyz="0.20231 0.000199 0.22543"
        rpy="0 0 0" />
      <mass
        value="0.44457" />
      <inertia
        ixx="0.000385"
        ixy="0"
        ixz="0"
        iyy="0.006223"
        iyz="0"
        izz="0.006502" />
    </inertial>
  </link>
  <joint
    name="LF_KFE"
    type="revolute">
    <origin
      xyz="0.308 0 0"
      rpy="0 0 0" />
    <parent
      link="LF_THIGH" />
    <child
      link="LF_SHANK" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="LF_FOOT">
    <inertial>
      <origin
        xyz="0.022126 -1.2357E-13 0.00049896"
        rpy="0 0 0" />
      <mass
        value="0.03332" />
      <inertia
        ixx="0.000014"
        ixy="0"
        ixz="0"
        iyy="0.000014"
        iyz="0"
        izz="0.000019" />
    </inertial>
  </link>
  <joint
    name="LF_ADAPTER"
    type="fixed">
    <origin
      xyz="0.308 0 0.23"
      rpy="0 0 0" />
    <parent
      link="LF_SHANK" />
    <child
      link="LF_FOOT" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="RF_HIP">
    <inertial>
      <origin
        xyz="-0.000688 0.104882 0.003776"
        rpy="0 0 0" />
      <mass
        value="2.923111" />
      <inertia
        ixx="0.004820"
        ixy="0"
        ixz="0"
        iyy="0.007868"
        iyz="0"
        izz="0.008241" />
    </inertial>
  </link>
  <joint
    name="RF_HAA"
    type="revolute">
    <origin
      xyz="0.427 -0.075 -0.0095"
      rpy="3.1416 1.5708 0" />
    <parent
      link="base" />
    <child
      link="RF_HIP" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="RF_THIGH">
    <inertial>
      <origin
        xyz="0.184747 3.2E-05 0.129056"
        rpy="0 0 0" />
      <mass
        value="4.216358" />
      <inertia
        ixx="0.006286"
        ixy="0"
        ixz="0"
        iyy="0.028156"
        iyz="0"
        izz="0.029242" />
    </inertial>
  </link>
  <joint
    name="RF_HFE"
    type="revolute">
    <origin
      xyz="0 0 0"
      rpy="-1.5708 0 0" />
    <parent
      link="RF_HIP" />
    <child
      link="RF_THIGH" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="RF_SHANK">
    <inertial>
      <origin
        xyz="0.202313 0.000199 0.225432"
        rpy="0 0 0" />
      <mass
        value="0.444569" />
      <inertia
        ixx="0.000385"
        ixy="0"
        ixz="0"
        iyy="0.006223"
        iyz="0"
        izz="0.006502" />
    </inertial>
  </link>
  <joint
    name="RF_KFE"
    type="revolute">
    <origin
      xyz="0.308 0 0"
      rpy="0 0 0" />
    <parent
      link="RF_THIGH" />
    <child
      link="RF_SHANK" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="RF_FOOT">
    <inertial>
      <origin
        xyz="0.022126 -4.4892E-13 -1.0375E-06"
        rpy="0 0 0" />
      <mass
        value="0.03332" />
      <inertia
        ixx="0.000014"
        ixy="0"
        ixz="0"
        iyy="0.000014"
        iyz="0"
        izz="0.000019" />
    </inertial>
  </link>
  <joint
    name="RF_ADAPTER"
    type="fixed">
    <origin
      xyz="0.308 0 0.23"
      rpy="0 0 0" />
    <parent
      link="RF_SHANK" />
    <child
      link="RF_FOOT" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="LH_HIP">
    <inertial>
      <origin
        xyz="-0.000688 0.10488 0.003776"
        rpy="0 0 0" />
      <mass
        value="2.9231" />
      <inertia
        ixx="0.004820"
        ixy="0"
        ixz="0"
        iyy="0.007868"
        iyz="0"
        izz="0.008241" />
    </inertial>
  </link>
  <joint
    name="LH_HAA"
    type="revolute">
    <origin
      xyz="-0.427 0.075 -0.0095"
      rpy="0 1.5708 0" />
    <parent
      link="base" />
    <child
      link="LH_HIP" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="LH_THIGH">
    <inertial>
      <origin
        xyz="0.184717 3.2E-05 0.129056"
        rpy="0 0 0" />
      <mass
        value="4.216358" />
      <inertia
        ixx="0.006286"
        ixy="0"
        ixz="0"
        iyy="0.028156"
        iyz="0"
        izz="0.029242" />
    </inertial>
  </link>
  <joint
    name="LH_HFE"
    type="revolute">
    <origin
      xyz="0 0 0"
      rpy="-1.5708 0 0" />
    <parent
      link="LH_HIP" />
    <child
      link="LH_THIGH" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="LH_SHANK">
    <inertial>
      <origin
        xyz="0.202313 0.000199 0.225432"
        rpy="0 0 0" />
      <mass
        value="0.444569" />
      <inertia
        ixx="0.000385"
        ixy="0"
        ixz="0"
        iyy="0.006223"
        iyz="0"
        izz="0.006502" />
    </inertial>
  </link>
  <joint
    name="LH_KFE"
    type="revolute">
    <origin
      xyz="0.308 0 0"
      rpy="0 0 0" />
    <parent
      link="LH_THIGH" />
    <child
      link="LH_SHANK" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="LH_FOOT">
    <inertial>
      <origin
        xyz="0.022126 -7.7827E-14 0.00049896"
        rpy="0 0 0" />
      <mass
        value="0.03332" />
      <inertia
        ixx="0.000014"
        ixy="0"
        ixz="0"
        iyy="0.000014"
        iyz="0"
        izz="0.000019" />
    </inertial>
  </link>
  <joint
    name="LH_ADAPTER"
    type="fixed">
    <origin
      xyz="0.308 0 0.23"
      rpy="0 0 0" />
    <parent
      link="LH_SHANK" />
    <child
      link="LH_FOOT" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="RH_HIP">
    <inertial>
      <origin
        xyz="0.000688 0.104882 0.003776"
        rpy="0 0 0" />
      <mass
        value="2.923111" />
      <inertia
        ixx="0.004820"
        ixy="0"
        ixz="0"
        iyy="0.007868"
        iyz="0"
        izz="0.008241" />
    </inertial>
  </link>
  <joint
    name="RH_HAA"
    type="revolute">
    <origin
      xyz="-0.427 -0.075 -0.0095"
      rpy="3.1416 1.5708 0" />
    <parent
      link="base" />
    <child
      link="RH_HIP" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="RH_THIGH">
    <inertial>
      <origin
        xyz="0.184747 3.2E-05 0.129056"
        rpy="0 0 0" />
      <mass
        value="4.216358" />
      <inertia
        ixx="0.006286"
        ixy="0"
        ixz="0"
        iyy="0.028156"
        iyz="0"
        izz="0.029242" />
    </inertial>
  </link>
  <joint
    name="RH_HFE"
    type="revolute">
    <origin
      xyz="0 0 0"
      rpy="-1.5708 0 0" />
    <parent
      link="RH_HIP" />
    <child
      link="RH_THIGH" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="RH_SHANK">
    <inertial>
      <origin
        xyz="0.202313 0.000199 0.225432"
        rpy="0 0 0" />
      <mass
        value="0.444569" />
      <inertia
        ixx="0.000385"
        ixy="0"
        ixz="0"
        iyy="0.006223"
        iyz="0"
        izz="0.006502" />
    </inertial>
  </link>
  <joint
    name="RH_KFE"
    type="revolute">
    <origin
      xyz="0.308 0 0"
      rpy="0 0 0" />
    <parent
      link="RH_THIGH" />
    <child
      link="RH_SHANK" />
    <axis
      xyz="0 0 1" />
      <limit
      effort="300"
      velocity="10"
      lower="-3.0"
      upper="3.0"/>
  </joint>
  <link
    name="RH_FOOT">
    <inertial>
      <origin
        xyz="0.022126 -1.6187E-13 -1E-06"
        rpy="0 0 0" />
      <mass
        value="0.03332" />
      <inertia
        ixx="0.000014"
        ixy="0"
        ixz="0"
        iyy="0.000014"
        iyz="0"
        izz="0.000019" />
    </inertial>
  </link>
  <joint
    name="RH_ADAPTER"
    type="fixed">
    <origin
      xyz="0.308 0 0.23"
      rpy="0 0 0" />
    <parent
      link="RH_SHANK" />
    <child
      link="RH_FOOT" />
    <axis
      xyz="0 0 0" />
  </joint>
</robot>
